### PR TITLE
adding comments functionality to each article selected

### PR DESCRIPTION
**Pull request by jm-w-a**
Adding comments functionality to each article selected

### CODE REVIEW

File: src/components/Comments.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+const Comments = () => {
+  const { article_id } = useParams();
+  const [currentComments, setCurrentComments] = useState([]);
+
+  useEffect(() => {
+    getArticleComment(article_id).then((comments) => {
+      setCurrentComments(comments);
+    });
+  }, [article_id]);
```
Comment (popatre) [inline src/components/Comments.jsx:15]: Think this would work without the article_id as a dependency.

File: src/components/Comments.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      <h4>comments</h4>
+      <ul>
+        {currentComments.map(
+          ({ author, body, comment_id, created_at, votes }) => {
+            return (
+              <li key={comment_id} className="article-comment">
+                <h5>User: {author}</h5>
+                <p>
+                  Comment: {body}
+                  <br />
```
Comment (popatre) [inline src/components/Comments.jsx:28]: might be nicer to use css to get the gap you need - but upto you

**Review by popatre — CHANGES_REQUESTED** [review]
- Good to see the axios requests extracted into their own folder - nice and reusable
- Lovely stuff creating a separate component for the comments; stops the components getting too big and bloated
- Good that we've got the comments for the correct article showing 👍
- Excellent work displaying the comments

- [ ] We need some loading feedback for a better UX - let the user know that something is happening/comments are being requested
- [ ] Would be good for the articles which have no comments to get a little messages saying so e.g. 'No comments to show - why not be the first to add one?'

**Review by robert112233 — APPROVED** [review]
Well done for implementing Jonathan's feedback, good to merge!

**Pull request merged**